resources/src/commands/buns.rs

```diff
@@ -5,6 +5,7 @@ use slack_hook::{PayloadBuilder, Slack};
 // if the text contains any form of "buns", it will respond with "You are buns".
 // This can be used as an example command when creating new commands for
 // the Slack bot.
+// TODO: respond with just a buns emoji to the message
```

Comment:
After making the change to `heart.rs` to be `emoji_reaction.rs` you can use that function here to add the buns reaction.

Reply:
Will do

Reply:
Will address in #41